File: week_2/9_light_censor_control.cpp
int cdsPin = 0;
int ledPin = 11;
int cdsValue = 0;
int ledValue = 0;

void setup() {
    pinMode(ledPin, OUTPUT);
}

void loop() {
    cdsValue = analogRead(cdsPin);
    ledValue = map(cdsValue, 0, 1023, 0, 255);
    digitalWrite(ledPin, ledValue);
    delay(10);
}

File: week_2/9_light_censor_control.py
from Arduino import Arduino
import time

cdsPin = 0
ledPin = 13

board = Arduino("9600", port = "COM3")
board.pinMode(ledPin, "OUTPUT")

while True:
    cdsValue = board.analogRead(cdsPin)
    if (cdsValue < 500):
        board.digitalWrite(ledPin, "HIGH")
    else:
        board.digitalWrite(ledPin, "LOW")
    time.sleep(0.01)
